src/components/Modal/__stories__/modal.stories.mdx

```diff
@@ -410,7 +410,7 @@ clicking outside. Use this variant in case of sensitive or important messages, a
             {/** Please follow the design guidelines when implementing custom title **/}
             <ModalHeader
               description={"Description text goes here"}
-              title={<LegacyEditableHeading type={LegacyEditableHeading.types.h2} placeholder={"Modal title"} />}
+              title={<EditableHeading type={EditableHeading.types.H2} value={"Modal title"} />}
```

Comment:
it behaves differently from the previous component. It should span to 100%

now:
<img width="615" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/6093192/c425ef5d-9b41-4a8a-8d62-a217df7a20ff">

before:
<img width="675" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/6093192/5eb8ff67-c12c-403e-ac5b-762c08808b29">

Reply:
That's intentional, the new spec defines it as an inline element
![Screenshot 2023-11-01 at 12 22 26](https://github.com/mondaycom/monday-ui-react-core/assets/18269880/a507c580-c124-4b6f-aa26-957e541801bd)

See https://github.com/mondaycom/monday-ui-react-core/pull/1727#discussion_r1378548622

Reply:
As long as it’s verified with Meirav, we’re good